contrib/core/actions/http.yaml

```diff
@@ -38,8 +38,14 @@ parameters:
   params:
     description: Query params to be used with the HTTP request.
     type: object
+  password:
+    description: Password for username.
+    type: string
   timeout:
     default: 60
     description: Timeout for the HTTP request.
     type: number
+  username:
```

Comment:
You don't need to repeat these parameters here. Actions inherit runner's parameters.